<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QDialog" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>290</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>290</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Search by phone</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>274</width>
     <height>84</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertical_layout">
    <item>
     <widget class="QLabel" name="no_results_label">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;No results.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item alignment="Qt::AlignBottom">
     <widget class="QLineEdit" name="phone_number_box">
      <property name="placeholderText">
       <string>Enter patient's phone number...</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="buttons_layout">
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="search_button">
        <property name="text">
         <string>Search</string>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignTop">
       <widget class="QPushButton" name="cancel_search_button">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
